README.md

```diff
@@ -9,15 +9,27 @@
 
 ## Overview
 
-This is an application that packages the UI, RPS, and MPS into a single executable for use in an enterprise environment.
+Console is an application that packages the Sample Web UI, RPS, and MPS into a single executable for use in an enterprise environment. Users can add activated AMT devices to access device information and device management functionality such as power control, remote keyboard-video-mouse (KVM) control, and more.

```

Comment:
We should be clear that Console is a re-implementation of RPS and MPS in Go that provides similar device management functionality in an enterprise environment.